### PR TITLE
Revert pinecone v4 support

### PR DESCRIPTION
**Pull request by baskaryan**
Revert 9d13dcd

### CODE REVIEW

File: tests/integration_tests/vectorstores/test_pinecone.py [hunk 1]
```diff
@@ -132,28 +146,68 @@ def test_from_texts_with_scores(self, embedding_openai: OpenAIEmbeddings) -> Non
...
             ids=ids_1,
             embedding=embedding_openai,
             index_name=index_name,
+            namespace=index_name,
         )
-        # wait for the index to be ready
-        time.sleep(20)
         index_stats = self.index.describe_index_stats()
+        assert index_stats["namespaces"][index_name]["vector_count"] == len(texts) * 2
         assert index_stats["total_vector_count"] == len(texts) * 2
```
Comment (jamescalam) [inline tests/integration_tests/vectorstores/test_pinecone.py:211]: I think can remove this line as was added in prev PR?

File: tests/integration_tests/vectorstores/test_pinecone.py
Comment (jamescalam) [inline tests/integration_tests/vectorstores/test_pinecone.py:1]: can remove `import time` as was added with prev PR

File: tests/integration_tests/vectorstores/test_pinecone.py (hunk 1 above)
Comment (baskaryan) [reply tests/integration_tests/vectorstores/test_pinecone.py:211]: this is from sep pr than one being reverted afaict

File: tests/integration_tests/vectorstores/test_pinecone.py
Comment (baskaryan) [reply tests/integration_tests/vectorstores/test_pinecone.py:1]: used in L225 (was added by a diff pr)

**Pull request merged**